imports/ui/Lobby.jsx

```diff
@@ -140,32 +174,76 @@ class Lobby extends React.Component {
     return null;
   }
 
+  renderPlayerForm() {
+    const actionsJoin = [
+      <RaisedButton
+        style={{display: 'flex', margin:'auto', height: 60}}
+        labelColor="white"
+        label="OK"
+        backgroundColor=  {Colors.primary}
+        keyboardFocused={true}
+        onTouchTap={this.joinGame.bind(this)}
```

Comment:
You're creating a function in a render function, define it as static method and as an arrow function and then just reference the function here.